QualifiedTeachersApi/tests/QualifiedTeachersApi.Tests/Jobs/JobCollection.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Xunit;
+
+namespace QualifiedTeachersApi.Tests.Jobs;
+
+[CollectionDefinition("Job", DisableParallelization = true)]
```

Comment:
Do we need this collection? Hopefully future job tests could run in parallel with the tests added here.